packages/devtools_shared/lib/src/memory/heap_sample.dart

```diff
@@ -60,25 +60,25 @@ class HeapSample {
 
   final bool isGC;
 
-  EventSample _memoryEventInfo;
+  EventSample? _memoryEventInfo;
```

Comment:
this is another case where you want to use `late` to avoid spurious nullable types and `!`

Reply:
How will a late variable be used with the ??= operator below?

Reply:
re-wrote these fields so that we don't need to use nullable types nor late